packages/plugins/typescript/document-nodes/lib/typescript-document-nodes.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
```

Comment:
Can we remove this one?

Reply:
Oh... I think it was some artifact from lerna.